### PR TITLE
ci: deploy example projects and link them to landingpage

### PR DESCRIPTION
**Pull request by MariaLStefan**
Closes #653 

- 3 new git workflows. Angular and Vue deployment works, React deployment contains bugs, could be fixed with latest vite update.
- New landingpage section for example-projects

### CODE REVIEW

File: packages/landingpage/src/components/IvxHeader.vue [hunk 1]
```diff
@@ -25,7 +25,7 @@
           {{ version.name }} {{ i === 0 ? "(latest)" : "" }}
         </ino-option>
       </ino-select>
-
+
```
Comment (BenPag) [inline packages/landingpage/src/components/IvxHeader.vue:28]: Can you revert these changes? The extra spaces are not needed I guess :)

File: packages/elements-react-example/src/shared/components/NavItemLink.tsx [hunk 2]
```diff
@@ -1,5 +1,8 @@
 import { Link, useMatch } from 'react-router-dom';
 import { InoIcon, InoNavItem } from '../InovexElements';
+// const InoIcon = import.meta.glob('./shared/InovexElements');
+// const InoNavItem = import.meta.glob('./shared/InovexElements');
```
Comment (BenPag) [inline packages/elements-react-example/src/shared/components/NavItemLink.tsx:4]: Why you comment this out?

File: .husky/_/husky.sh [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+  sh -e "$0" "$@"
+  exitCode="$?"
+
+  if [ $exitCode != 0 ]; then
+    echo "husky - $hook_name hook exited with code $exitCode (error)"
+    exit $exitCode
+  fi
+
+  exit 0
+fi
```
Comment (BenPag) [inline .husky/_/husky.sh:30]: Do you need this? I removed this file, because we do not use husky anymore.

**Review by BenPag — CHANGES_REQUESTED** [review]
Thank you for the integration 👍 Small things left ;)

Following is not part of your issue:
Recently I worked on the packages deployment process. Personally I try to optimize the wohle CI stuff. In this case I think we should discuss if we want to integrate the example projects deployment into the release flow. And check if we need an manual trigger for example project deployment.

File: .github/workflows/test.yml [hunk 4]
```diff
@@ -35,26 +35,6 @@ jobs:
...
-            packages/elements-react/dist/
-
-    - if: github.ref == 'refs/heads/master'
-      name: upload elements-angular artifact
-      uses: actions/upload-artifact@v2
-      with:
-        name: elements-angular
-        path:
-            packages/elements-angular/elements/dist/
+    #- if: github.ref == 'refs/heads/master'
```
Comment (BenPag) [inline .github/workflows/test.yml:38]: Do we want to upload build artifacts for non master branches? In which cases we need this?

File: packages/elements-react-example/vite.config.ts [hunk 5]
```diff
@@ -6,5 +6,14 @@ export default defineConfig({
   plugins: [react()],
   server: {
     open: '/'
-  }
+  },
+  base: '/example-projects/react/',
```
Comment (BenPag) [inline packages/elements-react-example/vite.config.ts:10]: Is this config working locally too?

File: .github/workflows/test.yml (hunk 4 above)
Comment (janivo) [reply .github/workflows/test.yml:38]: Whops, forgot to uncomment this one

File: packages/elements-react-example/vite.config.ts (hunk 5 above)
Comment (janivo) [reply packages/elements-react-example/vite.config.ts:10]: Yes it does

**Review by BenPag — APPROVED** [review]
